Common/Product/SharedProject/ProjectNode.cs

```diff
@@ -2456,8 +2456,10 @@ internal enum EventTriggering {
                     oldLogger.Dispose();
                 }
             } else {
-                ((IDEBuildLogger)this.BuildLogger).OutputWindowPane = output;
+                this.BuildLogger.OutputWindowPane = output;
             }
+
+            this.buildLogger.RefreshVerbosity();
```

Comment:
This new line is about the only one that looks significant here. If we're making escrow changes, why make all the other changes to this file at this point?